src/main/java/Deadline.java

```diff
@@ -0,0 +1,23 @@
+public class Deadline extends Task {
+
+    protected String by;
+    public Deadline(String description, String by) {
+        super(description);
+        this.by = by;
+    }
+
+    public static Deadline DeadlineCon(String description, String by) throws InvalidTaskCreationException {
+        if (description.equalsIgnoreCase("")) {
+            throw new InvalidTaskCreationException("OOPS!!! The description of a Deadline Task cannot be empty.");
+        } else if (by.equalsIgnoreCase("")) {
+            throw new InvalidTaskCreationException("OOPS!!! The deadline time of a Deadline Task cannot be empty.");
+        } else {
+            return new Deadline(description, by);
+        }
+    }
```

Comment:
It's great that the DeadlineCon method is relatively small. However, would you like to consider using JavaDoc to explain what the code does? Also, "DeadlineCon" is a method and should be in camel case. You may wish to rename the "Con" term to a full word.